python/vowpalwabbit/pyvw.py

```diff
@@ -516,6 +545,11 @@ def example(self, stringOrDict=None, labelType=pylibvw.vw.lDefault):
     def __del__(self):
         self.finish()
 
+    def init_python_reduction_task(self, actual_python_reduction):
```

Comment:
add warming of you should not use by yourself or remove- might move to inside createWithCustomPythonReduction?